test/test_property.py

```diff
@@ -147,6 +147,7 @@ def test_owned_object_singleton(self):
         annotation_uri = rdflib.URIRef('http://examples.org#annotation_property')
         cd.annotation = sbol.property.OwnedObject(cd, annotation_uri, sbol.Identified,
                                                   '0', '1', None)
+        self.assertIs(cd.annotation, None)
```

Comment:
Probably better to use `assertIsNone` to get a better error message on failure:

```suggestion
        self.assertIsNone(cd.annotation)
```

Reply:
AssertIsNone.  Of course that's a thing.  Done in f1daa93efdee55e3d559cf919e464a82099a833c